cibuildwheel/venv.py

```diff
@@ -110,10 +110,12 @@ def virtualenv(
     if use_uv:
         call("uv", "venv", venv_path, "--python", python)
     else:
-        virtualenv_app = _ensure_virtualenv(version)
+        virtualenv_app, virtualenv_version = _ensure_virtualenv(version)
         if pip_version is None:
             pip_version = _parse_pip_constraint_for_virtualenv(dependency_constraint)
-        additional_flags = [f"--pip={pip_version}", "--no-setuptools", "--no-wheel"]
+        additional_flags = [f"--pip={pip_version}", "--no-setuptools"]
+        if virtualenv_version < Version("20.31") or Version(version) < Version("3.9"):
```

Comment:
I haven't followed this bug closely, but would just 
```suggestion
        if virtualenv_version < Version("20.31"):
```
be sufficient?

Reply:
virtualenv 20.31 was released for 3.8 too. It still includes `wheel` for 3.8 and so therefore it is helpful to remove it with `--no-wheel`.

Reply:
gotcha